src/Dropdown.php

```diff
@@ -863,18 +863,43 @@ public static function dropdownIcons($myname, $value, $store_path, $display = tr
                         $values[$file] = $file;
                     }
                 }
+                $rand = mt_rand();
                 self::showFromArray(
                     $myname,
                     $values,
                     array_merge(
                         [
                             'value'                 => $value,
                             'display_emptychoice'   => true,
-                            'display'               => $display
+                            'display'               => $display,
+                            'noselect2'             => true, // we will instanciate it later
+                            'rand'                  => $rand,
                         ],
                         $options
                     )
                 );
+
+                global $CFG_GLPI;
+
+                // templates for select2 dropdown
+                $js = <<<JAVASCRIPT
+                $(function() {
+                    formatFormIcon{$rand} = function(icon) {
```

Comment:
Will prevent function to be declared in global scope.
```suggestion
                    const formatFormIcon = function(icon) {
```